packages/tinacms/src/admin/pages/CollectionUpdatePage.tsx

```diff
@@ -153,7 +148,7 @@ const RenderForm = ({
           <div className="mb-2">
             <span className="block text-sm leading-tight uppercase text-gray-400 mb-1">
               <Link
-                to={`/collections/${collection.name}${folder.parentName}`}
```

Comment:
@kldavis4 Everything in folder was empty because there is no `~` in the edit link